sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018-2020 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import Form, TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestSaleOrderSecondaryUnit(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
We are missing https://github.com/OCA/sale-workflow/pull/2742, could you please include it?